### ISSUE TITLE
Filenames with periods break on the command line

### ISSUE THREAD

**Issue by Jzarecta** [xonsh/xonsh#1148]
Been trying xonsh, seems files break when their names have periods:
```bash
mplayer Videos/Love.and.Hip.Hop.Atlanta.S05E10.HDTV.x264-CRiMSON.mkv.part 
MPlayer SVN-r37857 (C) 2000-2016 MPlayer Team
224 audio & 451 video codecs
do_connect: could not connect to socket
connect: No such file or directory
Failed to open LIRC support. You will not be able to use your remote control.

Playing Videos/Love..
File not found: 'Videos/Love.'
Failed to open Videos/Love..


Exiting... (End of file)
xonsh: subprocess mode: command not found: .Hip.Hop.Atlanta.S05E10.HDTV.x264-CRiMSON.mkv.part
```

**Comment by scopatz** [comment]
This is an and/or issue

**Comment by adqm** [comment]
There was [some discussion](https://gitter.im/scopatz/xonsh?at=57563bd945cf128e5f1d064b) in the Gitter.im room.  There are a couple of different approaches we can take here, but it's not clear which is the "winner".

**Comment by adqm** [comment]
In the meantime, wrapping long/complicated filenames in quotes is a workaround in the meantime.

**Comment by gforsyth** [comment]
I believe this has been resolved in 0.4.7 and on master.  Please reopen if the issue persists.  Thanks!
